BUILD.bpf

```diff
@@ -0,0 +1,24 @@
+COPTS = [
+    '-Iexternal/libbpf/include',
+    '-Iexternal/libbpf/include/uapi',
+    '-DCOMPAT_NEED_REALLOCARRAY',
+    '-fPIC',
+    '-fvisibility=hidden',
+    '-g',
+    '-O2',
+    '-Werror',
+    '-Wall',
+    '-c',
+]
+
+cc_library(
+    name = "bpf-lib",
```

Comment:
nit: it is called libbpf, I would call the target libbpf.

Reply:
will change it